Sources/App/Fastlane/SlackCommand+Fastlane.swift

```diff
@@ -37,7 +37,7 @@ extension SlackCommand {
             Parameters:
             - name of the target (as in the project)
             - `version`: version of the app
-            - `branch`: release branch name. Default is `release/<version>`
+            - `branch`: release branch name. Default is `release/<name.lowercase()>/<version>`
```

Comment:
why won't we use the same case in the branch names? I always found it confusing and leading to extra code also in the Fastlane to account for these differences.

Reply:
That's a very good question. I think it's mainly by convention that all our branch names are already all lowercased (`ilya/xxx`, not `Ilya/xxx`)...

But actually mixing cases between branches can lead to some git oddities, especially on case-insensitive filesystems (see [this slack discussion for an example](https://babylonhealth.slack.com/archives/G22EWKXQT/p1561726614311800), so if some people happen to use `NHS111` vs `Nhs111` vs `nhs111` it would generate trouble for most of us, so personally I don't mind the convention